src/git.js

```diff
@@ -1,6 +1,6 @@
-import util from "./utils";
-import path from "path";
-import semver from "semver";
+const util = require("./utils");
```

Comment:
he he, a lot of changes to get away from `import` and `export`, but I think it's good to increase the speed and get away from the runtime babel stuff 👍